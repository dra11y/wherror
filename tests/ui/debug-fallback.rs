use wherror::Error;

#[derive(Error, Debug)]
#[error(auto)]
pub enum MyError {
    First,
    Second,
}

fn main() {}
