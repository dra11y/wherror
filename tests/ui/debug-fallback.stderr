error: expected one of: string literal, `transparent`, `fmt`, `debug`
 --> tests/ui/debug-fallback.rs:4:9
  |
4 | #[error(auto)]
  |         ^^^^
